arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts

```diff
@@ -132,6 +132,24 @@
 	vddio-supply = <&pm8953_l6>;
 };*/
 
+&i2c_5 {
+	status = "okay";
+
+	led-controller@36 {
+		compatible = "ti,lm3697";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x36>;
+
+		led@1 {
+			reg = <1>;
+			led-sources = <0 1 2>;
+			label = "white:backlight";
```

Comment:
Please try replacing the label with
```
function = LED_FUNCTION_BACKLIGHT;
color = <LED_COLOR_ID_WHITE>;
```

Reply:
If I do that dtc gives me syntax error. Also according to Documentation/leds-class.rst and Documentation/devicetree/bindings/leds/leds-lm3697.txt(or at least from what I can tell), the label method is the proper way of doing it.

Reply:
`label` is deprecated, see https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/leds/common.yaml#L54-L60

You need to include `#include <dt-bindings/leds/common.h>` at the top for the function & color constants

Reply:
Ahh, now it compiled. Now I just have to test in my phone is it works and then push.

Thank you!

Reply:
It worked, so ig I'm resolving this conversation.

Once again, ty!

Reply:
Great :)